README.md

```diff
@@ -7,6 +7,28 @@
 [Inspektor Gadget](https://inspektor-gadget.io) Desktop App based on [wails](https://wails.io) and
 [Svelte](https://svelte.dev). Available for Windows, Mac and Linux.
 
+## Installation
+
+### macOS
+
+#### Homebrew Cask (Recommended)
+
+```bash
+brew install --cask inspektor-gadget/tap/ig-desktop
```

Comment:
I looked a bit into this and afaiu this would need a dedicated repo inside the inspektor-gadget org to work. If we would want to use this repo, it would need to be added as tap first. I'll do some testing outside of this project first.